lib/deps/fsp.js

```diff
@@ -24,11 +24,13 @@ const mkdirp = require('mkdirp');
  * Promisifed versions of Node's filesystem methods.
  */
 module.exports = {
+  ...fs,
```

Comment:
what's this for?

Reply:
Initially for debugging, as this now exposes all of `fs` apis in `fsp.js`. E.g. some of the methods are patched to be promisified, but you can still call all other methods. Will revert for now, because I am no longer using any non-promisified version.